packages/react/src/primitives/types/phoneNumberField.ts

```diff
@@ -4,13 +4,44 @@ import { SelectFieldProps } from './selectField';
 import { TextInputFieldProps } from './textField';
 
 export interface PhoneNumberFieldProps extends TextInputFieldProps {
+  /**
+   * @description
+   * Sets a hidden and accessible label for the country or region code selector
+   */
   countryCodeLabel?: string;
+
+  /**
+   * @description
+   * Sets the name used when handling form submission for the country or region code selector
+   */
   countryCodeName?: string;
+
+  /**
+   * @description
+   * Sets the default country or region code that will be selected on initial render
```

Comment:
We shouldn't use term 'country code' since some countries share the same dial code. Please update to user term dial code